components/Article.jsx

```diff
@@ -5,28 +5,42 @@ import { Link } from 'react-router-dom';
 
 
 
+
 const Article = ({article, setArticle}) => {
     const { articleId } = useParams();
+    const [vote, setVote] = useState(0)
     useEffect(() => {
         fetchArticlesByID(articleId).then(({article}) => {
           setArticle(article);
+          setVote(article.votes)
         });
       }, [articleId]);
 
       if (!article) {
         return <div>Loading...</div>;
     }
+     const handleVote = (voteType) => {
+        const updatedVotes = voteType === 'up' ? vote + 1 : vote - 1
+        setVote(updatedVotes)
+        setArticle({...article, votes: updatedVotes})
+     }
```

Comment:
So this does work and votes are optimistically rendered which is great! However, when I refresh the page, the votes are 0 again! Don't forget to get the votes updated in the backend too!